### DISCUSSION TITLE
Support for configuring locally hosted LLM servers

### DISCUSSION THREAD

**Discussion by amithkoujalgi** [JudiniLabs/code-gpt-docs#220] [Ideas]
Hello.

First of all, thank you for creating this plugin. Its amazing! 🤩

I was wondering if we could have support for configuring locally hosted LLM servers in different ways. For instance, I can have an [Ollama docker](https://hub.docker.com/r/ollama/ollama) setup running locally with an exposed port. This way, it helps me isolate the LLM server and yet fully utilise it capabilities.

Let me know what you think. Thanks!

**Comment by uptopoint** [comment]
Hi All, If anyone has solution to this problem. 
I want to setup a ollama base url as only one of my machines support running Ollama. Other machines have terrible performance issues. 
But in codegpt I cant setup base url to some other machine and I cant use ollama.

Anyone has any pointers in that if there is a way to do this
